src/menutemplate.js

```diff
@@ -17,7 +17,13 @@ var MenuTemplate = function () {
       submenu: [
       {
         label: 'About Kitematic',
-        selector: 'orderFrontStandardAboutPanel:'
```

Comment:
Not necessary, as it's commented out :P - I should delete it completely to prevent any further confusion.